java-client/src/integration-test/java/com/cumulocity/sdk/client/devicecontrol/DeviceControlIT.java

```diff
@@ -58,6 +58,11 @@
 @Slf4j
 public class DeviceControlIT extends JavaSdkITBase {
 
+    String SOME_AGENT = "SomeAgent";
+    String OTHER_AGENT = "OtherAgent";
+    String SOME_DEVICE = "SomeDevice";
+    String OTHER_DEVICE = "OtherDevice";
```

Comment:
I like more naming like  AGENT, AGENT_2, DEVICE, DEVICE_2. 
We may have OTHER, but I dont see value in SOME.